tools/test/devicesim/gnmi_target/defs.go

```diff
@@ -17,7 +17,11 @@ package main
 import (
 	"flag"
 
+<<<<<<< HEAD
```

Comment:
I assume this should not be part of the patch, correct?

Reply:
something screwed up in my fork